pkg/util/walkers/copying_test.go

```diff
@@ -0,0 +1,62 @@
+package walkers
```

Comment:
Can we have a copyright header? Should we add a check for new files in the CI?

Reply:
>Can we have a copyright header?
Fixed.

>Should we add a check for new files in the CI?

Perhaps.

Reply:
Created https://github.com/kinvolk/lokomotive/issues/58 for tracking.